mkemlog.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * mkemlog: creates emlog files
+ *
+ * This utility can be used to create emlog files with out having
+ * to lookup the major number of /dev/emlog
+ *
+ * This code is freeware and may be distributed without restriction.
+ *
+ * James Rouzier <rouzier@gmail.com>
+ * July 7, 2016
+ *
+ */
+
+#define _BSD_SOURCE
```

Comment:
I'm getting the following warning (with glibc-2.22):

    In file included from /usr/include/stdio.h:27:0,
                     from mkemlog.c:15:
    /usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
     # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"

On my machine, mkemlog.c compiles just fine without any `_SOURCE` macros defined.